html/om_oss.html

```diff
@@ -19,29 +18,91 @@
                 <input id="search" type="text" placeholder="Sök">
             </form>
 
-            <ul>
-                <li><a href="html/om_oss.html">Om oss</a></li>
-                <li><a href="html/konstnarsguiden.html">Konstnärsguiden</a></li>
-                <li><a href="#">Vanliga frågor</a></li>
-                <li><a href="html/kontakt.html">Kontakt</a></li>
-                <li>Meny</li> <!-- Drop down menu here -->
-            </ul>
-
-            <a href="#"><img src="#" alt="svenska">Sv</a>
-            <a href="#"><img src="#" alt="english">En</a>
+        <ul>
+          <li><a href="html/om_oss.html">Om oss</a></li>
+          <li><a href="html/konstnarsguiden.html">Konstnärsguiden</a></li>
+          <li><a href="#">Vanliga frågor</a></li>
+          <li><a href="html/kontakt.html">Kontakt</a></li>
+          <li>Meny</li>
+          <!-- Drop down menu here -->
+        </ul>
 
-            <a href="#">Logga in</a>
+        <a href="#"><img src="#" alt="svenska" />Sv</a>
+        <a href="#"><img src="#" alt="english" />En</a>
 
-        </nav>
+        <a href="#">Logga in</a>
+      </nav>
     </header>
 
     <main>
-
-        <ul class="breadcrumbs">
-            <li>Startsida / </li>
-            <li></li>
+      <section>
+        <h1>Konstnärsnämndens uppgifter</h1>
+        <p>
+          <strong>
+            Konstnärsnämnden är en statlig myndighet under regeringen. Inom
+            Regeringskansliet ansvarar Kulturdepartementet för Konstnärsnämnden
+            och konstnärspolitiken. Myndighetens övergripande uppgift är att
+            främja möjligheter för konstnärer att vidareutveckla sitt
+            konstnärskap, att främja konstnärlig utveckling och stödja
+            nyskapande kultur.
+          </strong>
```

Comment:
I don't know if this needs to be `<strong>` in this case. None of the other pages uses it for this part. 
I have a `<p>` that is `<strong>` in kontakt.html but only because it is a Corona notice :)

Reply:
Changing the stuff right now! I'll marge it after.